fast64_internal/oot/oot_level_writer.py

```diff
@@ -744,45 +763,47 @@ def ootProcessEmpties(scene, room, sceneObj, obj, transformMatrix):
     if obj.data is None:
         if obj.ootEmptyType == "Actor":
             actorProp = obj.ootActorProperty
-            addActor(
-                room,
-                OOTActor(
-                    getCustomProperty(actorProp, "actorID"),
-                    translation,
-                    rotation,
-                    actorProp.actorParam,
-                    None
-                    if not actorProp.rotOverride
-                    else (actorProp.rotOverrideX, actorProp.rotOverrideY, actorProp.rotOverrideZ),
-                ),
-                actorProp,
-                "actorList",
-                obj.name,
-            )
+            if actorProp.actorID != "None":
```

Comment:
``ootEnumActorID`` it filled with ``("None", f"{i} (Deleted from the XML)", "None")`` for every actors, since Blender saves the index of the list when you have an enum prop, you can change the identifier without breaking anything, so this will prevent a data loss if the user deletes an actor in the XML file

To answer your question, we're just checking if the ID is not "None" for that case when the user deleted the current Actor from the list

Reply:
Ah gotcha, makes sense. You should add a comment here with that explanation.